pkg/operator/status.go

```diff
@@ -87,10 +87,18 @@ func (c *Controller) syncStatus(cr *imageregistryv1.Config, deploy *appsapi.Depl
 		Message: "",
 		Reason:  "",
 	}
-	if deploy == nil {
+	if cr.Spec.ManagementState == operatorapiv1.Unmanaged {
+		operatorAvailable.Status = operatorapiv1.ConditionTrue
+		operatorAvailable.Message = "The registry configuration is set to unmanaged mode"
+		operatorAvailable.Reason = "Unmanaged"
+	} else if deploy == nil {
 		if e, ok := applyError.(permanentError); ok {
 			operatorAvailable.Message = applyError.Error()
 			operatorAvailable.Reason = e.Reason
+		} else if removed {
+			operatorAvailable.Status = operatorapiv1.ConditionTrue
+			operatorAvailable.Message = "The deployment is removed"
```

Comment:
Could we make this Message something like down below when we read:

"The registry is removed"